mautrix/bridge/config.py

```diff
@@ -102,6 +102,13 @@ def do_update(self, helper: ConfigUpdateHelper) -> None:
         copy("bridge.management_room_text.additional_help")
         copy("bridge.management_room_multiple_messages")
 
+        copy("bridge.encryption.allow")
+        copy("bridge.encryption.default")
+        copy("bridge.encryption.key_sharing")
```

Comment:
These copies should also be removed from the bridge-specific config upgraders

```suggestion
        copy("bridge.encryption.key_sharing.allow")
        copy("bridge.encryption.key_sharing.require_cross_signing")
        copy("bridge.encryption.key_sharing.require_verification")
```